pyproject.toml

```
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["setuptools>=80", "setuptools-scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "python-cvmfsutils"
```

For backwards compatiblity I didn't change it but I'd suggest changing this to:

```suggestion
name = "cvmfsutils"
```

---

Thanks for the contribution, Chris.  Can this change be made without changing the operation of the rpm?

---

I think this metadata only matters for `pip` and I presume we can assume there aren't currently any downstream dependencies, especially given python-cvmfsutils isn't uploaded to PyPI.

---

One thing to consider here would be to name it `cmvfs` instead of `cvmfsutils` to match the name of the import, what do you think?

---

No I think that's too generic of a name.  I wouldn't mind changing the name of the import.  I don't know that anybody is using this library outside of the utilities that are included in it.

---

> I don't know that anybody is using this library outside of the utilities that are included in it.

You know one person 😉

> I wouldn't mind changing the name of the import to cvmfsutils

That was going to be my other suggestion but obviously it's a far more invaisive change, though if you think that's fine I'll make the change.

---

I think that's a good change.  We'll change the version to 1.0 to indicate an incompatibility.  Maybe it's better as a separate PR from this one though.